recipes/getmail6/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "getmail6" %}
+{% set version = "6.18.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1dda163d20d7c67571d66234d1749e2209bca8cbcb7b944a2381837a82b88661
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+    - getmail --help
+    - getmail_fetch --help
+    - getmail_maildir --help
+    - getmail_mbox --help
+    - getmail-gmail-xoauth-tokens --help
```

Comment:
You'll may need entry_points for these, not sure how upstream define these.

Reply:
The issue is they just copy the shell scripts https://github.com/getmail6/getmail6/blob/master/getmail_fetch

Reply:
Maybe skip Windows then?

Reply:
Ok, done